Fix instances where nouns contain(s) or (s), for example app(s) (#7089)
mendix/docs: content/en/docs/developerportal/deploy/on-premises-design/ms-windows/_index.md

1. Open the IIS Manager and navigate to the website you want to manage.
2. In the **Features View**, double-click **URL Rewrite**.

> 3. In the **Actions** pane on the right side of the screen, click **Add rule (or rules)…** to add a new rewrite rule.

Reviewer: ```suggestion
3. In the **Actions** pane on the right side of the screen, click **Add rule(s)…** to add a new rewrite rule.
```

4. Enter the **Server variable name** *HTTP_X_FORWARDED_PROTO*.
5. Click **OK**.
6. Click **Back to Rules**.

> 7. Click **Add rule (or rules)…**.

Reviewer: ```suggestion
7. Click **Add rule(s)…**.
```

If HTTPS was configured at step 5.3 it is recommended to redirect all unencrypted HTTP traffic to HTTPS. To configure this, follow these steps:

> 1. Click **Add rule (or rules)…**.

Reviewer: ```suggestion
1. Click **Add rule(s)…**.
```
